requirements_bundles.txt

```diff
@@ -4,5 +4,5 @@
 # It's automatically installed when running npm run bundle
 
 # These can be removed when upgrading to Python 3.x
-importlib-metadata>=1.6  # remove when on 3.8
```

Comment:
@DebangshuB  why are we updating the lib version

Reply:
If we were to keep the version as is and rebuild the image, we'll get an error that says `'EntryPoints' object has no attribute 'get'`.
According to [exerror.com](https://exerror.com/entrypoints-object-has-no-attribute-get/),
> _"On October 4, 2022 importlib-metadata released importlib-metadata 5.0.0 and in version 5.0.0 They have Deprecated EntryPoints."_

The suggested fix is to lock the version to `4.13.0` or to upgrade python. I've done the former.